client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -258,13 +258,13 @@ module.exports = {
 
 	domainManagementAddGoogleApps( pageContext ) {
 		setTitle(
-			i18n.translate( 'Add G Suite' ),
+			i18n.translate( 'Add Google Apps' ),
```

Comment:
_This_ one is needed - it's the title of that view :) Only the one below is used for analytics - sorry if did not make it clear before.